README.md

```diff
@@ -158,7 +158,7 @@ Some caveats:
 - Scripts run in a job have access to all environment variables, but they do not have
   access to the app itself, so e.g. can't use `ebmbot.settings`; use the environment
   variables these are based on instead.
-- PYTHONPATH is set to the parent of the workspace directory, so scripts can access other
+- PYTHONPATH is set to the root directory of the application, so scripts can access other
   modules under `workspace`.
```

Comment:
change to just "so scripts can access other modules"?  Theoretically, they can now use any other modules in the repo (which might be useful later, as they should now be able to use e.g. settings)